src/main/java/Rei.java

```diff
@@ -0,0 +1,168 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Rei {
+    public static void main(String[] args) {
+        String logo = "  ____  _____ ___ \n"
+                + " |  _ \\| ____|_ _|\n"
+                + " | |_) |  _|  | | \n"
+                + " |  _ <| |___ | | \n"
+                + " |_| \\_\\_____|___|\n";
+
+        System.out.println("Annyeong! I'm\n" + logo);
+        System.out.println("What can I do for you?");
+        System.out.println("-----------YOU------------");
```

Comment:
I like the creativity here: the printlines demarcate clearly who's printing what, like a chat log.